docs/docs/en/guide/installation/pseudo-cluster.md

```diff
@@ -146,10 +146,29 @@ mysql> GRANT ALL PRIVILEGES ON dolphinscheduler.* TO '{user}'@'localhost' IDENTI
 mysql> flush privileges;
 ```
 
+The script above does not work for latest verisons (e.g. 8.0.27) of mysql, use the script below instead:
```

Comment:
I think we should not use this description. Just put ```Mysql 8``` in this area. And ```Mysql 5.6/5.7``` above.

Reply:
Sure, I will fix it.